lib/ddtrace/contrib/resque/resque_job.rb

```diff
@@ -7,9 +7,30 @@
 module Datadog
   module Contrib
     module Resque
+      # Automatically configures jobs with {ResqueJob} plugin.
+      module Job
+        def perform
+          if Datadog.configuration[:resque][:workers].nil?
+            job = payload_class
+            job.extend(Datadog::Contrib::Resque::ResqueJob) unless job.is_a? Datadog::Contrib::Resque::ResqueJob
+          end
+        ensure
+          super
+        end
+      end
+
       # Uses Resque job hooks to create traces
       module ResqueJob
-        def around_perform(*args)
+        # `around_perform` hooks are executed in alphabetical order.
+        # we use the lowest printable character that allows for an inline
+        # method definition ('0'), alongside our naming prefix for identification.
+        #
+        # We could, in theory, use any character (e.g "\x00"), but this will lead
+        # to unreadable stack traces that contain this method call.
+        #
+        # We could also just use `around_perform` but this might override the user's
+        # own method.
+        def around_perform0ddtrace(*args)
```

Comment:
Minor: Maybe
```suggestion
        def around_perform0_ddtrace(*args)
```

just to make it a little more readable?